src/footer.scss

```
@@ -0,0 +1,16 @@
+.footer {
+  padding-top: 140px;
+  text-align: center;
+
+  &__copyright {
+    margin: 36px 0 100px;
+    color: #BDBDBD;
+  }
+}
+
+.app {
+  margin-top: 60px;
+  &__inst {
+    padding: 0 45px;
+  }
+}
```

```suggestion
}

```

no newline at end of file